Multisig/UI/Transactions/TransactionDetails/LoadingTransactionDetailsViewModel.swift

```diff
@@ -37,7 +38,6 @@ class LoadingTransactionDetailsViewModel: ObservableObject {
     }
 
     func reload(transaction: TransactionViewModel) {
-        guard status != .loading else { return }
```

Comment:
In the end, we can get this back, because this condition will be holding

Reply:
ok